src/javascripts/application.mjs

```diff
@@ -104,3 +111,52 @@ if (campaignCookieBanner) {
     subtree: true
   })
 }
+
+const serviceNavigationHeader = document.querySelector(
+  '.govuk-service-navigation'
+)
+
+if (
+  serviceNavigationHeader.hasAttribute('data-govuk-service-navigation-init')
+) {
+  const subNavTreeParents = serviceNavigationHeader.querySelectorAll('li')
+
+  Array.from(subNavTreeParents).forEach((subNavTreeParent) => {
+    const link = subNavTreeParent.querySelector('a')
+
+    const template = link.querySelector('template')
+
+    if (template) {
+      const firstClone = template.content.cloneNode(true)
+
+      link.removeChild(template)
+
+      const newButton = document.createElement('button')
+      newButton.classList.add('app-navigation__list-toggle')
+      newButton.setAttribute('subitems-toggle-closed', true)
+      newButton.innerHTML = '▼'
```

Comment:
**suggestion** Use the `textContent` from the link to create the button name and hide the link on mobile.